CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ git checkout -b feature/some_feature
 git commit -a -m "Addresses issue #NNN".
 
 # Push to the remote repository, explicitly telling git that the remote repository you want to push to is "origin" (the foobar fork).
-git push --set-upstream origin feature/some_feature
+git push --set-upstream origin feature/999
```

Comment:
This no longer matches the example on line 19 - do you want it to say 999 up there as well?